src/tooltip-manager.ts

```diff
@@ -0,0 +1,137 @@
+'use babel';
+
+/// <reference path="marked/index.d.ts" />
+
+import {CompositeDisposable} from 'atom';
+import {RemoteEditorService} from 'polymer-analyzer/lib/editor-service/remote-editor-service';
+import * as marked from 'marked';
+
+class TooltipManager {
```

Comment:
I think all of this class's properties and methods can be made private, except for a `dispose` method. Just as documentation of how it's expected to be used.

(typescript's `private` is nice, it's just documentation and a compile-time warning, it doesn't do anything to stop someone from accessing the private thing)